sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -47,28 +49,18 @@ async function main() {
     throw new Error("Test ID returned as undefined.");
 
   // Uploading .jmx file to a test
-  const fileUploadResult = await client
-    .path("/loadtests/{testId}/files/{fileId}", testCreationResult.body.testId, fileId)
-    .put({
-      contentType: "multipart/form-data",
-      body: {
-        file: readStream,
-      },
-    });
+  const fileUploadResult = await (await beginUploadAndFileValidation(client, testId, "sample", readStream)).pollUntilDone();
```

Comment:
The double await in one line doesn't feel clean in a customer-facing sample. Could this be simplified into two lines?

Reply:
```suggestion
  const poller = await beginUploadAndFileValidation(client, testId, "sample", readStream);
  const fileUploadResult = await poller.pollUntilDone();
```

Reply:
Done.